api/v1alpha3/suite_test.go

```diff
@@ -22,9 +22,9 @@ import (
 
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/kubernetes/scheme"
-	"sigs.k8s.io/cluster-api/internal/envtest"
 	ctrl "sigs.k8s.io/controller-runtime"
-	// +kubebuilder:scaffold:imports
```

Comment:
I'm not familiar with kubebuilder - what was this tag doing?

Reply:
kubebuidler will add new imports if you generate new APIs or stuff at those tags.

I'll have to take a closer look, but it's usually possible to place them somwhere where they belong and are not moved by `gci`

Reply:
I restored all `// +kubebuilder:scaffold:imports`. I'm not sure it makes a lot of sense to have them though.

I don't think we currently use the kubebuilder CLI at all to create new API types or reconcilers. And if we would use it I'm not sure how valid the result would be (compared to just copy/paste/modify existing API types or reconcilers)